packages/components/src/components/modal/modal.tsx

```diff
@@ -190,8 +191,16 @@ export class Modal {
   openedChanged(newValue) {
     if (newValue === true) {
       this.open();
+      if (this.allowInjectingStyleToBody) {
+        // The following style will disable body from scrolling when modal is open
+        document.body.style.overflow = 'hidden';
+      }
     } else {
       this.close();
+      if (this.allowInjectingStyleToBody) {
+        // remove injected css style from body
+        document.body.style.overflow = null;
```

Comment:
…and set it back to its previous value, or `null` if there was nothing.

Reply:
yeah, sure! I will adjust them.